Doc/library/sys.rst

```diff
@@ -209,6 +209,26 @@ always available.
    yourself to control bytecode file generation.
 
 
+.. data:: bytecode_prefix
```

Comment:
While I agree there's value in following existing precedent (as with the choice of `_prefix` as the attribute suffix), I think in this case the precedent could be misleading, as it may give the impression that the interpreter will only ever read bytecode files from under that prefix, which isn't the case: the interpreter will still happily read precompiled bytecode files from any directory where Python source files may appear. What it won't do is read the local `__pycache__` subdirectories in those source directories - it will look for the bytecode *caches* under the parallel `pycache_prefix` tree instead.

That ambiguity doesn't exist for `PYTHONDONTWRITEBYTECODE`, as that setting clearly doesn't affect the way the interpreter *reads* bytecode files.

(That said, if we do agree to go with `sys.pycache_prefix`, we may subsequently decide that it's worthwhile to start supporting a `PYTHONUPDATEPYCACHE=0` setting, and only keep `PYTHONDONTWRITEBYTECODE` around as a backwards compatibility shim)

Reply:
Ok, works for me; I'll update to `pycache_prefix`.